### ISSUE TITLE
Use Shipyard to build and deploy KIND clusters

### ISSUE THREAD

**Issue by sridhargaddam** [stolostron/submariner-addon#272]
Shipyard provides common tooling like Makefiles, scripts etc and to deploy KIND clusters which can be used for development and e2e testing. Currently, submariner-addon repo uses its own files for building images and to deploying clusters. Adding Shipyard to submariner-addon allows us to have a consistent experience with other Submariner repos.

**Comment by skitt** [comment]
The build infrastructure used for Stolostron can’t run separate containers; this has been tried before, see https://github.com/stolostron/submariner-addon/pull/93

**Comment by dfarrell07** [comment]
Yeah this was basically the first thing we tried when we started working on the addon. :(

**Comment by nyechiel** [comment]
@sridhargaddam it seems like this one needs to be closed?
